### DISCUSSION TITLE
Unity plugin

### DISCUSSION THREAD

**Discussion by Norite-dev** [Genesis-Embodied-AI/genesis-world#178] [Ideas]
It would be awesome to have your physics in Unity via plugin. Like MuJoCo did

**Comment by hyp1998** [comment]
How did you do it

**Reply by Norite-dev** [reply]
did what?)

**Reply by hyp1998** [reply]
你怎么通过插件在unity里面实现

**Reply by Norite-dev** [reply]
I guess there are options. Mujoco did it with its own unity package with scripts that must be attached to gameObjects, and depends on its runtime library that launches simultaneously when you click "Play" button
